template/{{ project_name }}/noxfile.py.jinja

```diff
@@ -56,9 +56,10 @@ def run_user_tests(session):
 @nox.session(name="coverage")
 def run_coverage(session):
     """Run the coverage tests and generate an XML report."""
-    session.posargs.append("--cov=src/{{project_slug}}")
-    session.posargs.append("--cov-report=xml")
-    run_user_tests(session)
+    session.install("setuptools", silent=False)
```

Comment:
Are we sure we need to install `setuptools` here? Does we receive failures without it?

Reply:
Oops I didn't notice this, we certainly don't need setuptools. I'll remove it.

Reply:
@agriyakhetarpal I think we do need setuptools, what if the generated project has a setuptools backend?

Reply:
I see, we can keep it in that case :)